app/controllers/concerns/umrdr/works_controller_behavior.rb

```diff
@@ -4,6 +4,41 @@ module WorksControllerBehavior
     include Sufia::Controller
     include Sufia::WorksControllerBehavior
     include CurationConcerns::CurationConcernController
+       
+    # override curation concerns, add form fields values
+    def build_form
+    
+      super
+
+      date_range =  @form["date_coverage"].first
+      date_from_to = date_range.split('/')
+
+      start_dt = EDTF.parse(date_from_to[0])
+      end_dt =  EDTF.parse(date_from_to[1])
```

Comment:
`start_dt, end_dt = date_range.split('/')` would accomplish what is being done here.  Alternatively, if the "date_coverage" of the form is expected to be already filled in, it could be parsed directly to an edtf interval.  Then the `begin` and `end` methods of the interval could be called to yield start and end dates.